src/containers/Node/Node.tsx

```diff
@@ -2,207 +2,207 @@ import React from 'react';
 
 import {Tabs} from '@gravity-ui/uikit';
 import {Helmet} from 'react-helmet-async';
-import {Link, useRouteMatch} from 'react-router-dom';
+import {useRouteMatch} from 'react-router-dom';
 import {useQueryParams} from 'use-query-params';
 
-import {AutoRefreshControl} from '../../components/AutoRefreshControl/AutoRefreshControl';
-import {BasicNodeViewer} from '../../components/BasicNodeViewer';
+import {EntityPageTitle} from '../../components/EntityPageTitle/EntityPageTitle';
 import {ResponseError} from '../../components/Errors/ResponseError';
 import {FullNodeViewer} from '../../components/FullNodeViewer/FullNodeViewer';
-import {Loader} from '../../components/Loader';
+import {InfoViewerSkeleton} from '../../components/InfoViewerSkeleton/InfoViewerSkeleton';
+import {InternalLink} from '../../components/InternalLink';
+import {PageMetaWithAutorefresh} from '../../components/PageMeta/PageMeta';
 import routes from '../../routes';
 import {
     useCapabilitiesLoaded,
     useDiskPagesAvailable,
 } from '../../store/reducers/capabilities/hooks';
 import {setHeaderBreadcrumbs} from '../../store/reducers/header/header';
 import {nodeApi} from '../../store/reducers/node/node';
-import type {AdditionalNodesProps} from '../../types/additionalProps';
 import {cn} from '../../utils/cn';
 import {useAutoRefreshInterval, useTypedDispatch} from '../../utils/hooks';
 import {PaginatedStorage} from '../Storage/PaginatedStorage';
 import {Tablets} from '../Tablets';
 
 import type {NodeTab} from './NodePages';
-import {
-    NODE_PAGES,
-    OVERVIEW,
-    STORAGE,
-    STRUCTURE,
-    TABLETS,
-    getDefaultNodePath,
-    nodePageQueryParams,
-} from './NodePages';
+import {NODE_TABS, getDefaultNodePath, nodePageQueryParams, nodePageTabSchema} from './NodePages';
 import NodeStructure from './NodeStructure/NodeStructure';
+import i18n from './i18n';
 
 import './Node.scss';
 
 const b = cn('node');
 
 const STORAGE_ROLE = 'Storage';
 
-interface NodeProps {
-    additionalNodesProps?: AdditionalNodesProps;
-    className?: string;
-}
-
-export function Node(props: NodeProps) {
+export function Node() {
     const container = React.useRef<HTMLDivElement>(null);
 
     const dispatch = useTypedDispatch();
 
-    const match =
-        useRouteMatch<{id: string; activeTab: string}>(routes.node) ?? Object.create(null);
+    const match = useRouteMatch<{id: string; activeTab: string}>(routes.node);
+
+    // NodeId is always defined here because the page is wrapped with specific route Router
```

Comment:
Let's better add a straightforward check, so no changes in Router api could affect our code.

Reply:
Fixed